src/py42/sdk/queries/__init__.py

```diff
@@ -1,21 +1,22 @@
 from py42.sdk.queries.query_filter import FilterGroup
+from py42 import settings
 
 
 class BaseQuery(object):
     def __init__(self, *args, **kwargs):
         self._filter_group_list = list(args)
         self._group_clause = kwargs.get(u"group_clause", u"AND")
-        self.page_size = 10000
+        self.page_number = kwargs.get(u"page_number") or 1
```

Comment:
Updated this to properly set the value to 1 is `page_number` is `None`. This one shouldn't need separating the same as `page_size`.

Reply:
Yeah, mostly agree, except that alerts use `0` as their starting page num

Reply:
Sure, I looked at that and unless I'm mistaken, the changes to `BaseQuery` shouldn't affect `AlertQuery` because `self.page_number` is overridden in `AlertQuery`. 

Do you think we need to change this, or is it okay how it is now?

Reply:
Ah, that is correct. I think it is good the way it is. I think it is also good that you removed setting page num in the FileEventQuery because that could potentially override what it is set here.. And it will always get set in the BaseQuery.  However, there is a chance that some day the Base Query and File Event Query have different page nums.. That seems highly unlikely, but I supposed a simple safer guard can be to ensure `page_num > 0` in `FileEventQuery`.

Reply:
But actually, don't do that.... I think if a user passes in a negative or incorrect page num, the query should just explode.